front/components/actions/mcp/details/MCPRunAgentActionDetails.tsx

```diff
@@ -35,43 +37,49 @@ export function MCPRunAgentActionDetails({
       return queryResource.resource.childAgentId;
     }
     if (lastNotification) {
-      if (isRunAgentProgressOutput(lastNotification.data.output)) {
-        return lastNotification.data.output.childAgentId;
-      }
-      if (
-        isRunAgentChainOfThoughtProgressOutput(lastNotification.data.output)
-      ) {
-        return lastNotification.data.output.childAgentId;
+      const output = lastNotification.data.output;
+      if (isRunAgentProgressOutput(output)) {
```

Comment:
Same here and, `isRunAgentProgressOutput` is now a combination of the 2 events we have for run agent